pkg/datapath/iptables/iptables.go

```diff
@@ -828,6 +829,160 @@ func (m *IptablesManager) installForwardChainRules(ifName, localDeliveryInterfac
 	return nil
 }
 
+func (m *IptablesManager) installMasqueradeRules(prog, ifName, localDeliveryInterface,
+	snatDstExclusionCIDR, allocRange, hostMasqueradeIP string) error {
+	if option.Config.Masquerade && !option.Config.EnableBPFMasquerade {
```

Comment:
While we are here, to reduce nestedness, you could `if option.Config.Masquerade && option.Config.EnableBPFMasquerade { return nil }`.

Reply:
Makes sense, will do that.